### PR TITLE
[4.0] unused legacy rtl scss

### PR DESCRIPTION
**Pull request by brianteeman**
As far as I can tell none of the classes removed here are actually used.
code review

### CODE REVIEW

File: administrator/templates/atum/scss/template-rtl.scss [hunk 1]
```diff
@@ -21,61 +21,13 @@ ul {
...
-.sidebar-wrapper[data-hidden="1"] ~ .status {
-  margin-right: 0;
-}
-
-// com_cpanel
-.com_cpanel {
-  .card {
-    .list-group {
-      > .published {
-        border-right: 5px solid var(--success);
```
Comment (wilsonge) [inline administrator/templates/atum/scss/template-rtl.scss:55]: Strictly on code review. This looks like the counter part to https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/pages/_com_cpanel.scss#L86 so superficially it is being used (either that or both parts aren't being used :) )

**Comment by wilsonge** [comment]
On code review alone I think the published and unpublished *might* be being used. Agreed that the rest is not

**Comment by brianteeman** [comment]
Would be good if someone could find out where its being used as I couldnt find it.

**Comment by wilsonge** [comment]
I have to say I don't see it either. But in this case we should remove the LTR SCSS too rather than it being some halfway house

**Comment by brianteeman** [comment]
will update the pr

**Comment by brianteeman** [comment]
Updated as requested

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**